notfound/utils.py

```diff
@@ -25,6 +26,10 @@ def replace_uris(app, doctree, nodetype, nodeattr):
             if uri.startswith('../'):
                 uri = uri.replace('../', '')
 
+        if re.match('^https?://', uri):
```

Comment:
nit: re.match already match only from the beginning, `^` isn't necessary

Reply:
I think it's better to be explicit.